docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '2.1'
 services:
   kytos:
-    image: amlight/kytos:latest
+    image: amlight/kytos:mongo
```

Comment:
When it's not a draft anymore make sure to revert this local change here.

Reply:
Ok, keeping an eye on it.

Reply:
@Alopalao, thanks for pushing these last commits, excellent e2e coverage. 

The only thing remaining is this thread here, and also if you could post a last execution of the tests locally executing this test suite `TestE2EKytosAuth` that you've added for the record of this PR then we can merge this one too.

Reply:
Changed.